### PR TITLE
Prevent lambda colds starts

### LINKED ISSUE #18: Cold start on Lambda seems to trigger CORS errors
We have intermittent CORS failures, and the flow seems to be something like this:

1. Visit the store
2. Log in
3. The "open issues" section errors out with CORS errors
4. Reload the page
5. No CORS error; open issues load just fine

I have no proof, but my suspicion is that this is an issue with AWS Lambda "cold starts". I'm not sure if there's a for-real fix, but the smash-it-with-a-hammer fix is to keep the lamda warm.

Looks like Serverless has a plugin for this (see [this tutorial](https://serverless.com/blog/keep-your-lambdas-warm/)), so we just need to put this in place.

### PR DESCRIPTION
**Pull request by samovertonjr**
Fix #18 adding serverless-plugin-warmup to yml. 
Also adding a check in graphql/server to prevent the cold starts

### CODE REVIEW

File: src/graphql/server.js [hunk 1]
```diff
@@ -58,5 +58,10 @@ server.applyMiddleware({ app });
 const handler = serverless(app);
 
 export const graphql = async (event, context) => {
+  // Prevents Lambda cold starts
+  if (event.source === 'serverless-plugin-warmup') {
+    return callback(null, 'Lambda is warm!');
```
Comment (jlengstorf) [inline src/graphql/server.js]: There’s no `callback` defined, so I think we can just `console.log` instead.

Comment (DSchau) [reply src/graphql/server.js]: @samovertonjr yup! That's the API for Lambdas, callback is the 3rd argument.

Comment (samovertonjr) [reply src/graphql/server.js]: or
`export const graphql = async (event, context, callback) => {
     if (event.source === 'serverless-plugin-warmup') {
       console.log('Lambda is warm');`

Comment (DSchau) [reply src/graphql/server.js]: You can also do something like this, I believe.

```js
export const graphql = async (event, context) => {
  if (event.source === 'serverless-plugin-warmup') {
    return 'Lambda is warm'
  }
}
```

per [this article](https://aws.amazon.com/blogs/compute/node-js-8-10-runtime-now-available-in-aws-lambda/)

**Pull request merged**